Makefile

```diff
@@ -63,12 +62,10 @@ LICENSE_HEADER:
 .PHONY: license
 license: LICENSE LICENSE_HEADER
 	cd client && npm ci && npm run license:fix
-	cd docker && npm ci && npm run license:fix
 
 .PHONY: test-license
```

Comment:
let's move the license check up to root or at least put it back because we need it on docker code too